knowledge-base/chart-datatable-binding.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to use DataTable as source of data for the Chart
+description: How to convert DataTable to List and use it as data source for the Chart
+type: how-to
+page_title: How to use DataTable as data source for the Chart
+slug: chart-datatable-binding
```

Comment:
What is more likely for developers to search - "chart bind (to) datatable" or "chart datatable binding". Think about it and if needed, adjust the slug and the file name. I am specifically interested in the grammar form of "bind..."